qt_ui/windows/basemenu/ground_forces/QGroundForcesStrategy.py

```diff
@@ -31,6 +33,18 @@ def cheat() -> None:
             if not enemy_cp.captured:
                 layout.addWidget(QLabel(enemy_cp.name))
                 layout.addWidget(QGroundForcesStrategySelector(self.cp, enemy_cp))
+
+                layout.addWidget(QLabel("Delay movement until"))
+                self.delayed_start_spinner = QTimeEdit()
+                self.delayed_start_spinner.setMinimumTime(QTime(0, 0))
+                self.delayed_start_spinner.setDisplayFormat("T+hh:mm:ss")
+                self.delayed_start_spinner.timeChanged.connect(self.save_delayed_start)
+                self.delayed_start_spinner.setToolTip(
+                    "Forces will not move from Waypoint 1 until the specified time has elapsed."
```

Comment:
Players don't know what waypoint 1 is (I didn't until I saw the screenshot in this PR).